multi_gpu/four/submit_4.sub

```diff
@@ -0,0 +1,24 @@
+universe = vanilla
+
+log = job_4.log
+#
+executable = run_parallel.sh
+
+output = out_4.out
+error = err_4.err
+#
+should_transfer_files = YES
+when_to_transfer_output = ON_EXIT
+transfer_input_files = http://proxy.chtc.wisc.edu/SQUID/gpu-examples/Miniconda3-latest-Linux-x86_64.sh,../env.yml,model_parallel.py
+
+
+require_gpus = (Capability == 7.5) && (11.1 <= DriverVersion)
```

Comment:
Will this work with `Capability >= 7.5`?

Reply:
I haven't tested on with `Capability == 8.0`, but my reason behind pinning 7.5 was that when I profiled the process GPU utilization, I was achieving high utilization with the 2080s. I don't think we want to encourage anyone to test this code on newer cards because a) those cards are more relied upon by researchers and b) running this code on an A100 would severely under utilize the card. I can certainly test it on the A100 just to make sure it works, though.

Reply:
That's good reasoning. In that case, can you please add a short comment here in the submit files explaining why the Capability is pinned?  It is specific to this example, not a multi-GPU requirement in general.